sdk/src/manifest.rs

```diff
@@ -1966,19 +1976,19 @@ pub(crate) mod tests {
         let signer = temp_remote_signer();
 
         // Embed a manifest using the signer.
-        let (out_vec, _out_manifest) = manifest
+        let (out_vec, out_manifest) = manifest
             .embed_from_memory_remote_signed("png", image, signer.as_ref())
             .await
             .expect("embed_stream");
 
         // try to load the image
-        let manifest_store = Reader::from_stream("image/png", Cursor::new(out_vec)).unwrap();
+        let manifest_store = Reader::from_stream_async("image/png", Cursor::new(out_vec))
+            .await
+            .unwrap();
 
-        /* to be enabled later
-                // try to load the manifest
-                let mut validation_log = DetailedStatusTracker::new();
-                Store::from_jumbf(&out_manifest, &mut validation_log).expect("manifest_load_error");
-        */
+        // try to load the manifest
+        let mut validation_log = DetailedStatusTracker::new();
+        Store::from_jumbf(&out_manifest, &mut validation_log).expect("manifest_load_error");
```

Comment:
we should be able to get this result from the above Reader::from_stream_async result, no need to call again here.

Reply:
I just uncommented your tests that were disabled.  Feel free to change.

Reply:
I removed